docs/develop/sso-in-office-add-ins.md

```diff
@@ -11,15 +11,15 @@ Users sign in to Office (online, mobile, and desktop platforms) using either the
 
 ![An image showing the sign-in process for an add-in](../images/office-host-title-bar-sign-in.png)
 
-### Preview Status
+## Preview Status
 
 The Single Sign-on API is currently supported in preview only. It is available to developers for experimentation; but it should not be used in a production add-in. In addition, add-ins that use SSO are not accepted in [AppSource](https://appsource.microsoft.com).
 
 SSO requires Office 365 (the subscription version of Office). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features, including SSO, is turned off for that build.
 
 Not all Office applications support the SSO preview. It is available in Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets).
```

Comment:
We don't yet support Outlook due to flaky behavior I have encountered with Office.Runtime.Auth.getAccessToken.  SSO does "work" on Outlook but require fallback authentication.

Reply:
@TCourtneyOwen the file where this content exists (where you've added this comment) is actually the previously existing [SSO overview](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/sso-in-office-add-ins?branch=kb-sso-quickstart) article that's describing SSO in Office Add-ins at a high-level. The [SSO quick start](https://review.docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/sso-quickstart?branch=kb-sso-quickstart#configure-sso) that I've created (further down in this PR) doesn't mention support for Outlook, so I think we're good here.